GLTFSDK/Source/Serialize.cpp

```diff
@@ -676,28 +680,33 @@ namespace
     {
         rapidjson::Document::AllocatorType& a = document.GetAllocator();
         rapidjson::Value samplerValue(rapidjson::kObjectType);
+
         {
             RapidJsonUtils::AddOptionalMember("name", samplerValue, sampler.name, a);
 
-            if (sampler.magFilter != Sampler::kMagFilterDefault)
+            if (sampler.magFilter)
             {
-                samplerValue.AddMember("magFilter", sampler.magFilter, a);
+                samplerValue.AddMember("magFilter", sampler.magFilter.Get(), a);
```

Comment:
This behaviour seems to have changed. The previous code won't write out the value if it's default while this would.

Reply:
The old behaviour was actually incorrect. The real default for the property is 'not specified' rather than 'LINEAR'. If glTF SDK client code specifies any value at all then it should always be serialized